tests/e2e/specs/login-tests.js

```diff
@@ -59,67 +54,63 @@ describe("Successful Log in and log out", () => {
 
       cy.location("pathname").should("eq", "/dashboard");
     });
-  
 
     it("Should log out successfully", () => {
-      cy.contains("Log out")
-        .click();
+      cy.contains("Log out").click();
 
       cy.location("pathname").should("eq", "/logout");
```

Comment:
When I ran the test like this, it failed apparently because of a timeout issue, so I switched to a test of the page its self with a timeout. Does this pass consistently for you when you execute: 'nom run e2e', and click the test?

Reply:
I believe I was able to recreate the issue on my machine. I fetch the logout link now instead of trying to search and find the element containing the text log out. I will push the updated code tomorrow. When I submit my new PR, do you mind testing it out and see if it works on your computer? I just want to make sure. Thank you.

Reply:
@StewartWallace1115 @ti2ger92 any update here?

Reply:
@jkeat  @ti2ger92 Just updated it. Can someone pull and try it out, please?

Reply:
2 of 3 passed at first. After several reruns, more logouts failed. This is why I changed the success criteria for a completed logout from the url to an element on the page on my on PR, because that can have a timeout configured.

<img width="610" alt="Screen Shot 2020-01-08 at 8 28 07 PM" src="https://user-images.githubusercontent.com/19179199/72029932-839dfe80-3255-11ea-9752-c0a7b9e86ac4.png">

Reply:
@ti2ger92 Can you try it again when you a chance, please? If it fails, can you post a screenshot as you did before? Thank you.

Reply:
Hi! Works! Thanks Stewart for accommodating my Docker dreams.

I WAS able to get it to fail once like this:
1. fire everything up
2. run e2e test (pass)
3. immediately run test again (same failures as above).

However, once I gave the system a little time to settle, I got consistent positive tests. I say merge it!

Reply:
I ran into the same issue and had to increase the timeout from 1 second to 3 seconds to make it pass -- I can merge this for now, but I feel like in general, it'd be best not to rely on timeouts too much